packages/docs/docs/components/bezier.mdx

```diff
@@ -0,0 +1,295 @@
+---
+sidebar_position: 3
+slug: /bezier-curves
+---
+
+# Bézier Curves
+
+```tsx editor
+// snippet Cubic Bézier
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {CubicBezier} from '@motion-canvas/2d/lib/components';
+import {createRef} from '@motion-canvas/core/lib/utils';
+import {all} from '@motion-canvas/core/lib/flow';
+
+export default makeScene2D(function* (view) {
+  const bezier = createRef<Spline>();
+
+  view.add(
+    <CubicBezier
+      ref={bezier}
+      lineWidth={6}
+      stroke={'lightseagreen'}
+      p0={[-200, -70]}
+      p1={[120, -120]}
+      p2={[-120, 120]}
+      p3={[200, 70]}
+      end={0}
+    />,
+  );
+
+  yield* bezier().end(1, 1);
+  yield* bezier().start(1, 1).to(0, 1);
+});
+
+// snippet Quadratic Bézier
+import {makeScene2D} from '@motion-canvas/2d/lib/scenes';
+import {QuadBezier} from '@motion-canvas/2d/lib/components';
+import {createRef} from '@motion-canvas/core/lib/utils';
+import {all} from '@motion-canvas/core/lib/flow';
+
+export default makeScene2D(function* (view) {
+  const bezier = createRef<Spline>();
+
+  view.add(
+    <QuadBezier
+      ref={bezier}
+      lineWidth={6}
+      stroke={'lightseagreen'}
+      p0={[-150, 50]}
+      p1={[0, -120]}
+      p2={[150, 50]}
+      end={0}
+    />,
+  );
+
+  yield* bezier().end(1, 1);
+  yield* bezier().start(1, 1).to(0, 1);
+});
+```
+
+Bézier curves are ubiquitous in computer graphics. Motion Canvas comes with
+components to draw both quadratic as well as cubic Bézier curves.
+
+:::tip
+
+If you're trying to draw more complicated shapes than single Bézier curves allow
+for, check out the [`Spline`][spline] component, instead.
+
+:::
+
+## Using the components
+
+To draw Bézier curves, we can use the [`QuadBezier`][quad-bezier] component for
+quadratic Bézier curves and [`CubicBezier`][cubic-bezier] component for cubic
+Bézier curves, respectively.
+
+:::info
+
+Most of the examples on this page can be switched between cubic and quadratic
+Bézier curves by using the dropdown at the right side of the animation player.
+
+:::
```

Comment:
I think this could be shortened to:
```suggestion
Each example below is applicable to both the [`QuadBezier`][quad-bezier] and
[`CubicBezier`][cubic-bezier] nodes. You can switch between the two types of
curves using the dropdown on the right side of the animation player.
```